LayoutTests/http/wpt/fetch/fetch-as-blob.js

```diff
@@ -1,87 +0,0 @@
-if (self.importScripts) {
-  importScripts('/resources/testharness.js');
-}
-
-promise_test(function(t) {
```

Comment:
These tests in this file are already merged into the `fetch/api/body/mime-type.any.js` WPT tests so this one is removed in this PR.